src/DataCollection.UWP/Views/FeatureEditorView.xaml

```diff
@@ -25,6 +24,7 @@
                         <TextBlock Style="{StaticResource BaseTextBlockStyle}" Text="{Binding PopupFieldValue.Field.Label, Mode=OneWay}" />
                         <TextBox
                             AcceptsReturn="True"
+                            Margin="0,5,0,5"
```

Comment:
You could consider declaring `<Thickness>0,5,0,5</Thickness>` as a static resource and then re-using.  If we (or a user) wanted to tweak this in the future, then they could just do it in one place.